CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.1.1 - 2019-08-21
+This is a maintenance release.
+
+Bugfix:
+  * (SERVER-2602) Do not manage egd. For FIPS enabled builds explicitly set this to `/dev/random` via the `JAVA_ARGS` variable. FOSS users who would like to manage their egd source should use `JAVA_ARGS` as well.
+  * Fix deprecation warning to suggest using `Pkg::Util::Git.fail_on_dirty_source`.
```

Comment:
This is worded a little strangely. Maybe something more like 'Stop calling deprecated method `Pkg::Util::Version.fail_on_dirty_source`' ?

Reply:
sounds good. i wasn't sure the best way to word it

Reply:
alrighty i updated it